src/SFA.DAS.EAS.Employer_Account.Database/Tables/JobHistory.sql

```diff
@@ -0,0 +1,6 @@
+﻿CREATE TABLE [dbo].[JobHistory]
```

Comment:
What do you think to naming this `Jobs` for parity with the `Jobs` project name? If using EF this would allow for a `Jobs` DbSet of `Job` types.